tanjun/components.py

```diff
@@ -266,6 +276,7 @@ def copy(self: _ComponentT, *, _new: bool = True) -> _ComponentT:
             self._message_commands = list(commands.values())
             self._metadata = self._metadata.copy()
             self._names_to_commands = {name: commands[command] for name, command in self._names_to_commands.items()}
+            self._repeaters = [copy.copy(repeater) for repeater in self._repeaters] if self._repeaters else []
```

Comment:
repeater should have a "copy" method which is called here rather than using copy.copy